contracts/contracts/materializer/CrossMessenger.sol

```diff
@@ -0,0 +1,38 @@
+// SPDX-License-Identifier: MIT OR Apache-2.0
+pragma solidity ^0.8.23;
+
+import {FvmAddress} from "../structs/FvmAddress.sol";
+import {SubnetID, IPCAddress} from "../structs/Subnet.sol";
+import {IpcEnvelope, IpcMsgKind, CallMsg} from "../structs/CrossNet.sol";
+import {IGateway} from "../interfaces/IGateway.sol";
+import {EMPTY_BYTES, METHOD_SEND} from "../../contracts/constants/Constants.sol";
+
+/// This is a simple helper contract for Materializer to test cross messages.
+contract CrossMessenger {
```

Comment:
@karlem For this contract, is it possible to shift to `examples` folder? That's where we put sample/custom implementations. This contract is not really part of the protocol, it feels kind of strange to put it here though.